core/src/Dscp/Core/Foundation/Educator.hs

```diff
@@ -391,6 +399,18 @@ data CertificateIssuerInfo = CertificateIssuerInfo
     , ciiUrl  :: ItemDesc
     } deriving (Show, Eq, Generic)
 
+-- | Custom 'ToJSON' and 'FromJSON' instance because of website/url name difference
+instance ToJSON CertificateIssuerInfo where
+    toJSON CertificateIssuerInfo {..} = object
+        [ "name"    .= ciiName
+        , "website" .= ciiUrl
+        ]
+
+instance FromJSON CertificateIssuerInfo where
+    parseJSON = withObject "CertificateIssuerInfo" $ \v -> CertificateIssuerInfo
+        <$> v .: "name"
+        <*> v .: "website"
+
```

Comment:
Heh, @Heimdell is going to enjoy all this (he's splitting this module into submodules in a parallel PR).
Btw, why was it needed?

Reply:
You could simply rename `cliUrl` to `cliWebsite` and use `deriveJSON`

Reply:
Yeah I was a bit confused because in some example/doc `cliUrl` was filled with an email I think.

I'll `deriveJSON` with `cliWebsite` and put it in `Dscp.Core.Aeson`